Paramedic additions

TacoLlama opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Adds several new items for paramedics, and a seniority uniform and PDA

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
More content for paramedic players, and some changes I think just make sense
## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="210" height="179" alt="image" src="https://github.com/user-attachments/assets/28da937e-43aa-4449-8730-8b9b06c50aa8" /> ![PDA](https://github.com/user-attachments/assets/fafe329e-24e5-4131-ace9-ca768b2c17cb)![Senior PDA](https://github.com/user-attachments/assets/4e6f46f7-84d1-4f57-b724-7350888da181) ![Monitor](https://github.com/user-attachments/assets/f7992669-41a7-48f8-bd80-a19bebb234e6)


<img width="1392" height="418" alt="image" src="https://github.com/user-attachments/assets/d679a411-06ec-4933-85ca-de8ac657683b" />

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->

:cl: TacoLlama
- add: Paramedic defibrillator
- add: Lockable body bags
- add: Added a rescue knife for paramedics so the clown spider you find dragging the dead body away has something to worry about
- add: Senior Paramedic uniforms and PDA
- tweak: Modified the standard paramedic PDA's visuals
- tweak: Added infection resistance to void suit, medical gloves, and sterile masks
- tweak: Resprite of the emergency crew monitor
- tweak: Added External access to Paramedic IDs

TacoLlama committed:
Paramedic defib

TacoLlama committed:
Add infection resistance to medical masks, gloves, and the Paramedic voidsuit

TacoLlama committed:
Lockable body bags
because it is SO annoying when the random tider opens your body bag to see if there is loot inside

TacoLlama committed:
Paramedic Rescue knife
for the maints spiders

TacoLlama committed:
fix rescue knife

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

rylandsuper commented:
interesting

ScorchFollower commented:
A knife that does blunt? Interesting however a lockable via ID bodybag is very cool.

TacoLlama commented:
> A knife that does blunt? Interesting however a lockable via ID bodybag is very cool.

it's switchable like the boxcutter, so it does blunt when closed and slash when open

ScorchFollower commented:
> > A knife that does blunt? Interesting however a lockable via ID bodybag is very cool.
> 
> it's switchable like the boxcutter, so it does blunt when closed and slash when open

Ahh, never mind. I see now, I was looking at it when it wasn't activated lol. My bad.

TacoLlama committed:
Senior Paramedic uniforms and PDA

TacoLlama committed:
Setting up loadouts and making some small changes and fixes

TacoLlama committed:
forgot to put the knife on the whitelist

TacoLlama committed:
move the knife from the bag to the belt
yayy commit spam

TacoLlama committed:
Handheld Monitor
Will probably change how exactly this works later but that's a problem for tomorrow me!

TacoLlama committed:
Fix linter and increase useRate

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama committed:
see if this fixes things

TacoLlama committed:
add ContainerContainer component to defib

Happyrobot33 commented:
needs to be resolved with https://github.com/ss14Starlight/space-station-14/pull/2273

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama committed:
Remove paramedic monitor and replace emergency monitor sprite
SO much better than mine

TacoLlama committed:
Add external access to paramedic
why don't they have it? EVA is a good portion of paramedic work

TacoLlama commented:
> needs to be resolved with #2273

Resolved

Conflee reviewed (approved):
sprites approved

TacoLlama committed:
Fix ContainerContainer comp
I used it wrong (of course) (let's hope this works or I might cry)

TacoLlama committed:
add cell slot to containercontainer

TacoLlama committed:
very tiny change to the senior paramedic uniform
when I say tiny I mean three pixels

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama commented:
oops

TacoLlama commented:
hold on I messed up the conflicts

TacoLlama committed:
maybe fix my mistakes? maybe create more problems?

Happyrobot33 reviewed (changes requested):
needs starlight comments for upstream files

TacoLlama committed:
fixing missing starlight comments

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

Happyrobot33 commented:
test fail valid

TacoLlama commented:
> test fail valid

how are those related to this PR? the jani one isn't and the other one is... mapping? not actually sure what that is

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

TacoLlama committed:
Add inhands for monitor, from SeamLesss

TacoLlama committed:
deleted a line when resolving merge conflicts accidentally

TacoLlama committed:
Merge branch 'starlight-dev' into paramedic-additions

wonderfulnewworld commented:
i yearn somebody please merge this

KillerTheFlareon commented:
why does the knife do structural damage
